api/api_ethereum.go

```diff
@@ -1129,15 +1131,21 @@ func (args *EthTransactionArgs) setDefaults(ctx context.Context, b Backend) erro
 				if b.ChainConfig().IsEthTxTypeForkEnabled(head.Number) {
 					// TODO-Klaytn: Klaytn is using fixed BaseFee(0) as now but
 					// if we apply dynamic BaseFee, we should add calculated BaseFee instead of params.ZeroBaseFee.
-					fixedGasPrice.Add(fixedGasPrice, new(big.Int).SetUint64(params.ZeroBaseFee))
+					gasPrice.Add(gasPrice, new(big.Int).SetUint64(params.ZeroBaseFee))
 				}
-				args.GasPrice = (*hexutil.Big)(fixedGasPrice)
+				args.GasPrice = (*hexutil.Big)(gasPrice)
 			}
 		}
 	} else {
 		// Both maxPriorityFee and maxFee set by caller. Sanity-check their internal relation
-		if args.MaxFeePerGas.ToInt().Cmp(args.MaxPriorityFeePerGas.ToInt()) < 0 {
-			return fmt.Errorf("maxFeePerGas (%v) < maxPriorityFeePerGas (%v)", args.MaxFeePerGas, args.MaxPriorityFeePerGas)
+		if isKIP71 {
+			if args.MaxFeePerGas.ToInt().Cmp(gasPrice) < 0 {
+				return fmt.Errorf("maxFeePerGas (%v) < BaseFee (%v)", args.MaxFeePerGas, gasPrice)
+			}
+		} else {
+			if args.MaxFeePerGas.ToInt().Cmp(args.MaxPriorityFeePerGas.ToInt()) < 0 {
```

Comment:
I think this can be used for `isKIP71` condition also.
You can add validation to check `args.MaxPriorityFeePerGas.ToInt().Cmp(gasPrice) != 0 || args.MaxFeePerGas.ToInt().Cmp(gasPrice) != 0`.

Reply:
We need to discuss this. Please ignore this for now.